yajsapi/runner/index.ts

```diff
@@ -695,9 +697,17 @@ export class Engine {
           promise_timeout(10),
         ]);
 
+        let done_workers = new Set([...workers].filter((worker) => !worker.isPending()));
+
         workers = new Set([...workers].filter((worker) => worker.isPending()));
         services = services.filter((service) => service.isPending());
 
+        for (let worker_task of [...done_workers]) {
+            if (worker_task.isPending()) {
+                await worker_task;
+            }
+        }
```

Comment:
This does nothing: `done_workers` is the set of workers for which `worker.isPending()` is false, so the condition in `if` is never satisfied (assumming a worker cannot change from `! isPending()` to `isPending()`).

Reply:
Removed.